index.html

```diff
@@ -0,0 +1,165 @@
+                        </div>
+
+                        <div class="g-card-body">
+                            <img src="./images/program_icon_02.png" alt="Jump Suit" />
+                            <h5 class="card-title">Jump Suit</h5>
+                            <p class="card-text">Jumpsuit are raining dresses and your clients needs to have lots of Jumpsuit. Give them the spark that.</p>
+                          
+                        </div>
+
+                        <div class="g-card-body">
+                            <img src="./images/program_icon_05.png" alt="Native Wear"/>
+                            <h5 class="card-title">Native Wears</h5>
+                            <p class="card-text">Don't let your client miss that OWAMBE. Here in Globerry we give you the best for your client.</p>
+                          
+                        </div>
+
+                        <div class="g-card-body">
+                            <img src="./images/program_icon_04.png" alt="Bridals" />
+                            <h5 class="card-title">Bridals</h5>
+                            <p class="card-text">Bridal wears to suit your wedding and all kinds of event is taught here. Come Learn</p>
+                           
+                        </div>
+                        
+                        <div class="g-card-body">
+                            <img src="./images/program_icon_03.png" alt="Wedding Dress" />
+                            <h5 class="card-title">Wedding Dress</h5>
+                            <p class="card-text">Wedding, Wedding, Wedding. Make your wedding dress count. We sew and you love your wedding.</p>
+                           
+                        </div>
+                    </div>
+                      
+                </div>
+                
+            </div>
+              
+        </div>
+        <h5 class="text-center bottom-push">Traning Schedule (Download Here)</h5>
+    </section>
+
+
+    <section class="featured_speakers">
+        <h3 class="text-center">Featured Designers</h3>
+
+        <div class="container">
+            <div class="grid-container">
+
+            </div>
+        </div>
+    </section>
+
+
+
+    <section class="featured_partners">
+        <h3 class="text-center">Partners</h3>
+        <div class="container">
+            <img src="./images/logo1.webp" class="partners" alt="Partners" />
+            <img src="./images/logo2.webp" class="partners"  alt="Partners" />
+            <img src="./images/logo3.jpg" class="partners" alt="Partners" />
+            <img src="./images/logo6.png" class="partners" alt="Partners"  />
+            <img src="./images/logo4.jpg" class="partners" alt="Partners" />  
+        </div>
+    </section>
+
+
+    <footer class="footer_section">
+        <div class="container">
+            <div class="row">
+                <div class="col-4">
+                    <a class="" href="#">Globerry Couture</a>
+                </div>
```

Comment:
- [ ] Nice footer. I recommend putting the app logo in place of the app name as it is on the original design 
**imp**
![image](https://user-images.githubusercontent.com/37901469/169524763-f3c1f0e2-739b-4573-9e52-06fdc9543b74.png)

- [ ] Also as it is on original design, please kindly change the background color of the footer to white when we are on mobile view
**imp**
![image](https://user-images.githubusercontent.com/37901469/169525241-1d6359b0-9edc-4c8f-afae-d6bcd77a4b4a.png)

**Design**
![image](https://user-images.githubusercontent.com/37901469/169525329-dc6e5bf1-3044-4ccf-94a9-7353b0dbdea4.png)

Reply:
Dear Caleb,

The background color for the desktop  is white as seen below from the design
![image](https://user-images.githubusercontent.com/46789910/169547798-a9b3ea64-d44f-4ce3-9982-930e66132a85.png)

My Design Desktop
![image](https://user-images.githubusercontent.com/46789910/169547904-650ed31b-d7f6-41a7-8eee-13c6621a40b6.png)

From Design (Mobile)
![image](https://user-images.githubusercontent.com/46789910/169547990-24a6e46f-9912-4280-9997-c35d799f9e15.png)

My Design Mobile
![image](https://user-images.githubusercontent.com/46789910/169548054-e14768f5-6888-4761-a952-1ef6de5f526c.png)

Maybe you might want to throw more light to this or am i missing something. Thanks